### PR TITLE
Add visitOnClock cypress command to allow for draft testing

### PR DESCRIPTION
**Pull request by paulfalgout**
Shortcut Story ID: [sc-41419]

### CODE REVIEW

**Comment by paulfalgout** [comment]
Still has some coverage issues

File: test/integration/globals/default-route.js
Comment (nmajor25) [inline test/integration/globals/default-route.js:1]: There are a few more instances of `.visit('/')` in the tests. Do we want to remove those as well?

A few in these files:

* `/patients/worklist/reduced-schedule.js`
* `/patients/sidebar/filter-sidebar.js`
* `/globals/app-nav.js`

**Review by nmajor25 — APPROVED** [review]
This is great 👍

File: test/integration/globals/default-route.js
Comment (paulfalgout) [reply test/integration/globals/default-route.js:1]: yeah might as well.  those were never needed.. actually

**Comment by paulfalgout** [comment]
This coveralls report vs local is driving me nuts.. just going to go with this and work on the coverage later.

**Pull request merged**